recipes/knot/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: knot
+  version: "1.0.0"
+
+source:
+  fn: KNOT-1.0.0.tgz
+  url: http://mathbio.nimr.mrc.ac.uk/download/KNOT/KNOT-1.0.0.tgz
+  md5: 2556876109bae506fe171a45902db0b6
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - system
```

Comment:
You are right. I checked the `system` package, and it's definitely not what I thought it was. My reason for including it was that the `${PREFIX}/bin` and `${PREFIX}/log` folders don't get created automatically if there are no dependencies. However, that's better fixed with `mkdir -p`.